test/tools/TestExe/TestExe.cs

```diff
@@ -36,6 +36,10 @@ private static int Main(string[] args)
                     case "-stderr":
                         Console.Error.WriteLine(args[1]);
                         break;
+                    case "-stderrandout":
+                        Console.WriteLine(args[1]);
+                        Console.Error.WriteLine(args[1]);
```

Comment:
seems like it would be better if stderr had different output to stdout.  Maybe just reverse the text content for stderr?

Reply:
Makes sense! Added